### DISCUSSION TITLE
Importing a .pfd file

### DISCUSSION THREAD

**Discussion by slazar394** [e2nIEE/pandapower#2433] [Q&A] [answered]
I have a .pfd file for a power system exported from DIgSILENT PowerFactory. I've went through the documentation and saw that there are two options for importing the system to pandapower. However, they both rely on having PowerFactory installed. Is there a way to import the .pfd file without PowerFactory?

**Comment by vogt31337** [comment, marked as answer]
@slazar394 , currently not.
